src/serac/physics/state/finite_element_vector.hpp

```diff
@@ -370,4 +370,13 @@ double min(const FiniteElementVector& fe_vector);
  */
 double norm(const FiniteElementVector& state, const double p = 2);
 
+/**
+ * @brief Find the L2 norm of the error of a finite element vector with respect to an exact solution
+ *
+ * @param state The state variable to compute a norm of
```

Comment:
it looks like this description of `state` was copied from `norm()` and needs to be updated. I never know what to put down for doxygen descriptions of arguments like this, almost everything ends up sounding tautological.

Reply:
Thanks. I just updated it. I agree, but tautological is better than wrong. In a way, it's is a good sign, because it means the function is probably self-documenting.